buildSrc/src/main/java/com/uber/okbuck/core/manager/GroovyManager.java

```diff
@@ -22,17 +24,20 @@
 
   public static final String GROOVY_HOME_LOCATION =
       OkBuckGradlePlugin.WORKSPACE_PATH + "/groovy_installation";
+  public static final String GROOVY_HOME = "groovy_home";
+  public static final String GROOVY_HOME_TARGET = "//" + GROOVY_HOME_LOCATION + ":" + GROOVY_HOME;
 
   private final Project rootProject;
-  @Nullable private Set<String> dependencies;
+  @Nullable private String groovyVersion;
+  @Nullable private Set<ExternalDependency> dependencies;
 
   public GroovyManager(Project rootProject) {
     this.rootProject = rootProject;
   }
 
   @SuppressWarnings("ResultOfMethodCallIgnored")
   public void setupGroovyHome() {
-    String groovyVersion = GroovySystem.getVersion();
+    groovyVersion = GroovySystem.getVersion();
```

Comment:
Does this statical method contains any state? Or is coupled to any lifecycle? If not, groovy version could be not nullable and defined entirely at line 31

Reply:
seems like it is static in ReleaseInfo.java. https://github.com/groovy/groovy-core/blob/master/src/main/org/codehaus/groovy/util/ReleaseInfo.java#L79

will move it out to a static property.

Reply:
done